README.md

```diff
@@ -138,6 +138,37 @@ As soon as that line of code is reached in any of the running instances of your
 
 ![Breakpoint Hit](doc/images/breakpoint-hit.png?raw=true)
 
+Note that the directory layout of the code that is being debugged does not have to exactly match the source code specified in the Debug UI.  In particular, if a snapshot is specified in a file in the Debug UI, a breakpoint is set in the deployed file with the longest matching path suffix.
+
+An example will help illustrate this.  That is, suppose that the code on the running instance is structured as follows:
+```
+/running/instance/package.json
+/running/instance/main.js
+/running/instance/a/index.js
+/running/instance/b/a/index.js
+```
+Now suppose on your local machine you have a copy of the code as follows, and you have selected the `/local/machine` directory in the Debug UI as the directory containing the code to debug.
+```
+/local/machine/package.json
```

Comment:
It would be better to talk about repo paths rather than absolute paths so that this applies more generally to source in github etc. Concretely, I would recommend omitting the `/local/machine` from the prefix here and talking about 'Now suppose that your source code repository is structured as follow...'